### PR TITLE
feat: export imported types

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes KILTProtocol/ticket#2697

Re-exports all the types we import in the types package.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: packages/types/src/Address.ts [hunk 1]
```diff
@@ -5,10 +5,9 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import type { Prefix } from '@polkadot/util-crypto/address/types'
-import type { HexString } from '@polkadot/util/types'
 import '@polkadot/keyring' // TS needs this for the augmentation below
-import type { KeyringPair } from './index.js'
+
+import type { HexString, KeyringPair, Prefix } from './index'
```
Comment (arty-name) [inline packages/types/src/Address.ts:10]: Should this be consistent with the rest?
```suggestion
import type { HexString, KeyringPair, Prefix } from './Imported'
```

**Review by arty-name — APPROVED** [review]
Should other SDK packages import these types from the types package?

File: packages/types/src/Address.ts (hunk 1 above)
Comment (rflechtner) [reply packages/types/src/Address.ts]: good spot, I wonder if that happened due to the merge. fixed in 0f58cd5df20f67db2b300a3a0e28f2a16762fe25

**Comment by rflechtner** [comment]
> Should other SDK packages import these types from the types package?

Good point!

**Pull request merged**